.gitignore

```diff
@@ -158,3 +158,7 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+
+# VScode settings
```

Comment:
its useful to share vscode settings via git for things like debug configuration and linter/format settings so it's best to not gitignore these